qa/L0_perf_kaldi/test.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+# Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#  * Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+#  * Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#  * Neither the name of NVIDIA CORPORATION nor the names of its
+#    contributors may be used to endorse or promote products derived
+#    from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+# Test with 20.03 for now because model is compatible with that version
+REPO_VERSION="20.03"
+
+git clone ssh://git@gitlab-master.nvidia.com:12051/dl/JoC/asr_kaldi.git && \
+    cd asr_kaldi && \
+    ./scripts/docker/build.sh && \
+    ./scripts/docker/launch_download.sh && \
+    ./scripts/docker/launch_server.sh
+
+CONCURRENCY=2000
+
+# Client only supports GRPC (5 iterations on the dataset)
+./scripts/docker/launch_client.sh -i 5 -c ${CONCURRENCY} >> client_1.log 2>&1
```

Comment:
Is the client in the docker image something similar to our perf_client?

Reply:
Similar yes but they claim that the cannot use perf_client instead. There is a related discussion in the ticket.

Reply:
Thanks for the pointer!